### PR TITLE
Fix substations properties column

### PR DESCRIPTION
**Pull request by flomillot**
Remove the custom cell renderer and format into a basic string instead

### CODE REVIEW

**Review by EstherDarkish — APPROVED** [review]
Code OK Tests OK

**Review by anistouri — CHANGES_REQUESTED** [review]
There are 2 tooltips on the "Properties" column, the custom tooltip and the one from AGGRID

**Comment by flomillot** [comment]
> There are 2 tooltips on the "Properties" column, the custom tooltip and the one from AGGRID

Ah yes indeed the custom one appear after 3 seconds. I see with the PO which one we keep.

**Review by anistouri — APPROVED** [review]

File: src/components/spreadsheet/utils/config-tables.js [hunk 1]
```diff
@@ -104,11 +114,9 @@ export const TABLES_DEFINITIONS = {
             {
                 id: 'Properties',
                 field: 'properties',
-                cellRendererSelector: () => {
-                    return { component: SensiProperties };
-                },
-                tooltipComponent: CustomTooltipKeyValue,
-                tooltipValueGetter: toolTipValueGetterProperties,
+                valueGetter: propertiesGetter, // valueFormatter does not work here
+                // tooltipComponent: CustomTooltipKeyValue, // TODO uncomment / delete with US 2326
```
[the remark this answers is not in the document]
Comment (flomillot) [reply src/components/spreadsheet/utils/config-tables.js:118]: Yes you're right, I do it

**Pull request merged**